godot-core/src/builtin/string/godot_string.rs

```diff
@@ -29,12 +31,12 @@ impl GodotString {
         Self { opaque }
     }
 
-    ffi_methods! {
-        type sys::GDExtensionStringPtr = *mut Opaque;
-
-        fn from_string_sys = from_sys;
-        fn from_string_sys_init = from_sys_init;
-        fn string_sys = sys;
+    /// Returns a 32-bit integer hash value representing the string.
+    pub fn hash(&self) -> u32 {
```

Comment:
This name may shadow the [`Hash::hash(&mut H)`](https://doc.rust-lang.org/std/hash/trait.Hash.html#tymethod.hash) method from the standard library. Should probably be fine though, as the trait can be invoked with fully-qualified syntax.